src/richie/apps/core/templates/richie/web_analytics.html

```diff
@@ -0,0 +1,16 @@
+{% block web_analytics %}
+    {% if WEB_ANALYTICS_ID %} 
+        {% if not WEB_ANALYTICS_PROVIDER or WEB_ANALYTICS_PROVIDER == "google_analytics" %}
```

Comment:
WEB_ANALYTICS_PROVIDER has a default value so this extra check is counter productive?
```suggestion
        {% if WEB_ANALYTICS_PROVIDER == "google_analytics" %}
```